TodoReview.py

```diff
@@ -150,6 +150,8 @@ def run(self, edit, **args):
 		window = self.view.window()
 		paths = args.get('paths', None)
 		settings = Settings(self.view, args.get('settings', False))
+		if not settings.get('current_file', False):
+			paths = [self.view.file_name()]
```

Comment:
Thanks for the hints, that was really helpful. I wonder why my first version even worked :).
I made the check for the `current_file` setting top priority and if the file was not saved I print a note to the console and leave the function.
I left the rest of the logic (`open_files_only` etc) untouched and just moved it into the `else` branch of the `current_file` check.